_stylesheets/base/_typography.scss

```
@@ -71,3 +70,7 @@ span {
   padding-right: 8px;
   padding-left: 3px;
 }
+
+div.highlight {
+  font: $text-monospace;
```

The [font property](https://www.w3schools.com/cssref/pr_font_font.asp) is a shorthand that lets you define multiple properties in one line. Can you update this to be more explicit and use `font-family` since that is all we are changing here?

```suggestion
  font-family: $text-monospace;
```